blog/rome-incentives/README.md

```diff
@@ -0,0 +1,225 @@
+# Post
+
+#### Title
+
+The Rome Incentive Structure
+
+#### Purpose
+
+Inform our community about Rome's incentive structure.
+
+#### url
+
+blog.joystream.org/rome-incentives/
+
+#### Cover
+
+<p align="center"><img src="rome-incentives.png"></p>
+
+#### Lead
+
+Rome offers a new incentive structure for our testnet participants.
+
+#### Body
+
+As with Sparta, Athens and Acropolis we plan to pay users in monero (XMR) to participate on our upcoming testnet release, codenamed Rome. Our fifth testnet ambitiously introduces two new paid roles which we are extremely excited about!
+
+Executive Summary
+-----------------
+
+- `Content Creators` receive up to $0.60 per qualifying item of content uploaded\
+- `Content Curators` earn either $30 or $60 per week, depending on workload\
+- `Storage Providers` will share $300 per week, plus $0.025/GB per week\
+- `Validators` are competing for their share of $50 per week\
+- `Council Members` are paid $10 for getting elected, with a $10 bonus\
+- `Bug Reporters` can continue to earn monero for reporting and/or fixing bugs
+
+Who, Why, What and How?
+-----------------------
+
+As with previous incentivized testnets, anyone who wants to can participate on Rome and earn monero, but there is some extra help and guidance for those that ask for it in our [Telegram group](https://t.me/JoyStreamOfficial).
```

Comment:
Add that the faucet will only include smaller payouts, whereas tlgrm, twtr and mailing list subs can ask for tokens...